map2loop/thickness_calculator.py

```diff
@@ -199,6 +211,7 @@ def __init__(self):
         """
         self.thickness_calculator_label = "InterpolatedStructure"
         self.lines = []
```

Comment:
Add the interpolated orientations

Reply:
Also can we keep track of the thickness estimates for the locations on the map

Reply:
Will add this as a feature request for the Interpolated structure thickness calculator.

Reply:
The interpolated orientations are stored as the "dip" column in the `self.lines` object.
 
Added an extra object called `self.location_tracking` that contains the points coordinates and the thickness estimates.